### ISSUE TITLE
[0.12.1] python 2.6: ImportError: No module named backports.functools_lru_cache

### ISSUE THREAD

**Issue by ppolewicz** [crsmithdev/arrow#517]
b2-command-line-tool travis builds started crashing recently with arrow 0.12.1 on python 2.6, while https://pypi.python.org/pypi/arrow claims that 0.12.1 should work with 2.6 both in metadata and readme:

```pytb
======================================================================
ERROR: Failure: ImportError (No module named backports.functools_lru_cache)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/build/Backblaze/B2_Command_Line_Tool/test/test_b2http.py", line 18, in <module>
    from b2.b2http import _translate_and_retry, _translate_errors, B2Http, ClockSkewHook
  File "/home/travis/build/Backblaze/B2_Command_Line_Tool/b2/b2http.py", line 13, in <module>
    import arrow
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/arrow/__init__.py", line 3, in <module>
    from .arrow import Arrow
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/arrow/arrow.py", line 19, in <module>
    from arrow import util, locales, parser, formatter
  File "/home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/arrow/parser.py", line 12, in <module>
    from backports.functools_lru_cache import lru_cache  # pragma: no cover
ImportError: No module named backports.functools_lru_cache
```
it would seem that 2.6 was forgotten here:
https://github.com/crsmithdev/arrow/pull/504/files#diff-2eeaed663bd0d25b7e608891384b7298R44

@ramonsaraiva

**Comment by Syeberman** [comment]
Hi @ppolewicz!  We are in the process of removing 2.6 as a supported version: https://github.com/crsmithdev/arrow/pull/472.  2.6 hasn't received an update [in five years](https://www.python.org/download/releases/2.6.9/), and even 2.7 is [close to retiring](https://pythonclock.org/).

Unfortunately, this leaves you in a tight spot.  If you're unable to update to Python 2.7, you may have to version-lock arrow to an older version.  If none of those options are possible, we would consider a PR that fixes any 2.6 incompatibilities...but as we've also dropped 2.6 from our testing it is quite likely to break again.

**Comment by ppolewicz** [comment]
@Syeberman I am all for dropping 2.6, but releasing a 2.6-incompatible version which is announced in pip text and metadata as 2.6-compatible doesn't seem right - this gives pip no way to get a version of arrow that would be appropriate.

Could you please fix the PR I mentioned and release a new version OR fix the pypi metadata and reupload the package to pypi, so that 0.12.1 is not announced as 2.6-compatible?
